packages/core/src/components/expandable/expandable.scss

```diff
@@ -19,4 +18,8 @@
   visibility: inherit;
 }
 
+:host(:not([is-open])) {
```

Comment:
Dit attribuut wordt met logica in de Anime callback functions gezet. Het komt erop neer dat `is-open` pas wordt gezet als de animatie _naar_ open toe klaar is. Bij het sluiten wordt `is-open` verwijderd bij het begin van de animatie.